packages/cardhost/app/components/card-creator.js

```diff
@@ -1,9 +1,18 @@
 import { action } from '@ember/object';
+import { dasherize } from '@ember/string';
 import CardManipulator from "./card-manipulator";
 
 export default class CardCreator extends CardManipulator {
   @action
   updateCardId(id) {
     this.card = this.data.createCard(`local-hub::${id}`);
   }
+
+  get sanitizedType() {
+    return this.selectedField.type.replace(/::/g, '/').replace(/@/g, '');
+  }
+
+  get fieldEditor() {
+    return `fields/${dasherize(this.sanitizedType)}-editor`;
+  }
```

Comment:
I don't think this is being used anywhere. Remove this

Reply:
this is used in teh field renderer--we use a component helper to get the editor for a field.

Reply:
this looks like a dupe of the field renderer code. agreed on removing.